bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqBindingConfig.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.myq.internal;
+
+import org.openhab.core.binding.BindingConfig;
+
+/**
+ * This represents the configuration of a openHAB item that is binded to garage
+ * door opener. It contains the following information:
+ *
+ * <ul>
+ * <li>The Device number of the device returned from the MyQ AP (0 would be the first device)</li>
+ * <li>The type is how the binding uses the item</li>
+ * <ul>
+ * <li>Switch</li>
+ * <li>StringStatus</li>
+ * <li>ContactStatus</li>
+ * </ul>
+ * <li>id is unused at this time</li> </ul>
+ * 
+ * @author Scott Hanson
+ * @since 1.8.0
+ */
+public class MyqBindingConfig implements BindingConfig {
+	/**
+	 * The binding type of the myq item.
+	 * <ul>
+	 * <li>Switch</li>
+	 * <li>Rollershutter</li>
+	 * <li>StringStatus</li>
+	 * <li>ContactStatus</li>
+	 * </ul>
+	 */
+	public enum ITEMTYPE {
+		Switch, Rollershutter, StringStatus, ContactStatus
+	};
+
+	int deviceIndex;
+	ITEMTYPE type;
```

Comment:
Instead of an `ITEMTYPE` enum, just keep the `Class<? extends Item>` of the item, so you're not creating an enum that partially mirrors the various item types in openHAB.